core/dom/node.js

```diff
@@ -267,7 +267,7 @@ CKEDITOR.tools.extend( CKEDITOR.dom.node.prototype, {
 	 * @returns {CKEDITOR.dom.node} The node being inserted.
 	 */
 	insertBefore: function( node ) {
-		node.$.parentNode.insertBefore( this.$, node.$ );
+		node.$.parentNode.insertBefore( CKEDITOR.tools.legacyUnsafeHtml(this.$), node.$ );
```

Comment:
should we move this to the other PR?

Reply:
Removed this as it was just a testing artifact. insertBefore takes a DOM Node object so it the conversion is unnecessary here